portfolio/src/main/webapp/index.html

```diff
@@ -62,11 +62,14 @@ <h1>The Portfolio</h1>
       </p>
       <p></p>
       <strong>Chonky Cats:</strong>
-      <img src="images/1800x1200_cat_relaxing_on_patio_other.jpg" alt="Cat" style="width:600px;height:400px;">
+      <img src="images/1800x1200_cat_relaxing_on_patio_other.jpg" alt="Cat" 
+      style="width:600px;height:400px;">
```

Comment:
nit: indent this style line 2 spaces.

Since it is not its own element, but an extension of the previous, it should be indented somewhat.

According to this: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping, you should indent 4, so probably do 4 instead.

This makes the distinction between 2, of a child element, and 4 for a continuation of a previous element.  Which probably makes it easier to understand and see the difference.

The most important thing, is that you have a style, and stick with it.